mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -1410,7 +1410,7 @@ void searchShouldNotReturnNoMoreDelegatedMailboxes() throws Exception {
     @Nested
```

Comment:
I would like to have at least the following test:

```
USER 1 creates a mailbox and save a mail in it
Using a systemSession with a different username, I am able to read it
```

Maybe also demonstrating basic operations can be nice too (rename mbx, delete mbx, create mbx, copy msg, setFlags, delete msg, move msg, read msg, search mbx, search msg)

Reply:
Hmmm... regarding basic operations it depends. Some might be correct, but not all (I looked at createMailbox for example, it asserts that the MailboxPath belongs to the user, nothing to do with rights)

Reply:
That is where having some tests allow us to better understand the limits of this `systemSession` stuff and thus is valuable